src/main/resources/stock_trades_schema.avro

```diff
@@ -0,0 +1,64 @@
+{
+  "namespace": "ksql",
+  "name": "StockTrade",
+  "doc": "Defines a hypothetical stock trade using some known simulated stock symbols.",
+  "type": "record",
+  "fields": [
+    {
+      "name": "side",
+      "doc": "A simulated trade side (buy or sell or short)",
+      "type": {
+        "type": "string",
+        "arg.properties": {
+          "options": [ "BUY", "SELL", "SHORT" ]
+        }
+      }
+    },
+    {
+      "name": "quantity",
+      "doc": "A simualted random quantity of the trade",
+      "type": {
+        "type": "int",
+        "arg.properties": {
+          "range": {
+            "min": 1,
+            "max": 5000
+          }
+        }
+      }
+    },
+    {
+      "name": "symbol",
+      "doc": "Simulated stock symbols",
+      "type": {
+        "type": "string",
+        "arg.properties": {
+          "options": [ "ZJZZT","ZVZZT","ZWZZT" ]
+        }
+      }
+    },
+    {
+      "name": "price",
+      "doc": "A simulated random trade price in pennies",
+      "type": {
+        "type": "int",
+        "arg.properties": {
+          "range": {
+            "min": 5,
+            "max": 1000
+          }
+        }
+      }
+    },
+    {
+      "name": "account",
```

Comment:
1. Not clear what "account" represents
2. For consideration: instead of account, how about [user id](https://github.com/confluentinc/kafka-connect-datagen/blob/0.1.x/src/main/resources/pageviews_schema.avro#L13-L18)??  That also opens up potential to run this datagen schema in parallel with the [users](https://github.com/confluentinc/kafka-connect-datagen/blob/0.1.x/src/main/resources/users_schema.avro#L18) one, so you can do joins later in the stream processing application.  (Note the format of user id)

Reply:
Account is very standard for trading here and a logical aggregate for both buy side and sell side use cases in finance.  I could also add a user, but account applies well to future stream processing use cases in this field

Reply:
Goal is to leave door open for joins with another pre-bundled schema, so how about instead of replacing account, we add user?

Reply:
Added